tests/test_plots.py

```diff
@@ -332,6 +333,59 @@ def test_plot_mcse_models(self, datatree, datatree2, backend):
         assert "model" in pc.aes["color"].dims
         assert "/x" in pc.aes.groups
 
+    def test_plot_pair(self, datatree, backend):
```

Comment:
either here or in follow up tests we should make sure that `triangle` argument is taken into account. That is, with triangle both `scatter` only has `None` on the diagonal, with lower on the diagonal and above... Same thing for `marginal=False`